### ISSUE TITLE
New league messed up the layout.

### ISSUE THREAD

**Issue by kilpsz** [Knicklichtjedi/PoE_Trade_CSS#1]
https://i.imgur.com/d7VtIDC.png

**Comment by Knicklichtjedi** [comment]
Yeah, I had to wait for u/regularPoEplayer to update the CSS file.
Should be fixed now, but you need to reinstall the script. I added an Install link in the Readme that should update future versions automatically!
